budgetportal/webflow/urls.py

```diff
@@ -21,5 +22,15 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^infrastructure-projects/provincial/(?P<id>\d+)-(?P<slug>[\w-]+)/csv-download$",
+        views.ProvInfaProjectCSVDownload.as_view(),
+        name="provincial-infra-project-detail-csv-download",
+    ),
     url(r"^api/v1/", include(router.urls)),
+    url(
+        r"^api/v1/infrastructure-projects/provincial/search/csv-download/<uuid:uuid>",
+        views.ProvInfraProjectSearchViewCSVDownload.as_view(),
```

Comment:
decided to make it here not as a view set as its just one get action - let me know if it does make sense